DOCUMENTATION.md

```diff
@@ -0,0 +1,137 @@
+
+#### Filesystem
+
+List of filesystem subsets available to the application.
+
+Name | `flatpak override` equivalent | Type | Description
+--- | --- | --- | ---
+All filesystem files | `--filesystem=host` and `--nofilesystem=host` | Toggle | Permit (`--filesystem=host`) or prohibit (`--nofilesystem=host`) read-write access to the application to the whole filesystem. If it is permitted, the application has read-write access to every file and folder owned by you, the user. The rest will be read-only.
+All system libraries, executables and static data | `--filesystem=host-os` and `--nofilesystem=host-os` | Toggle | Permit (`--filesystem=host-os`) or prohibit (`--nofilesystem=host-os`) read-write access to the application to system libraries located in `/usr`. Since this directory requires root access to write, the permission will be read-only.
+All system configurations | `--filesystem=host-etc` and `--nofilesystem=host-etc` | Toggle | Permit (`--filesystem=host-etc`) or prohibit (`--nofilesystem=host-etc`) read-write access to the application to system configurations located in `/etc`. Since this directory requires root access to write, the permission will be read-only.
+All user files | `--filesystem=home` and `--nofilesystem=home` | Toggle | Permit (`filesystem=home`) or prohibit (`--nofilesystem=home`) read-write access to the application to the user directory (`$HOME`).
+Other files | `--filesystem=[PATH]`, `--filesystem=[PATH]:ro` and `--nofilesystem=[PATH]` | Input | Permit (`--filesystem=[PATH]`) or prohibit `--nofilesystem=[PATH]` read-write access to the application to the directory you desire, for example in Flatseal, you would put `~/games` (`--filesystem=~/games` using `flatpak override`) in the box if you want read-write access to `~/games`. If you want read-only access (`ro`), then you want append `:ro`, for example in Flatseal, you would put `~/games:ro` (`--filesystem=~/games:ro` using `flatpak override`) if you want read-only access to `~/games`.
+
+#### Persistent
+
+List of the homedir-relative paths created in the sandbox.
+
+Name | `flatpak-override` equivalent | Type | Description
+--- | --- | --- | ---
+Files | `--persist=[FILENAME]` | Input | If the application doesn't have access to the real homedir, make the (homedir-relative) path FILENAME a bind mount to the corresponding path in the per-application directory, allowing that location to be used for persistent data. This updates the [Context] group in the metadata. This option can be used multiple times. 
+
+#### Environment
+
+List of variables exported to the application.
+
+Name | `flatpak-override` equivalent | Type | Description
+--- | --- | --- | ---
+Variables | `--env=[VAR]=[VALUE]` | Input | Set an environment variable in the application. This updates the [Environment] group in the metadata. This overrides to the Context section from the application metadata. This option can be used multiple times. 
```

Comment:
I think having:

> Set an environment variable in the application.

Should be enough. The rest is a bit too technically and not strictly necessary.

Reply:
https://github.com/tchx84/Flatseal/pull/136/commits/68fff6c0526d5ea819169cd9ac057342e3f04d14